src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/AttributeExtensionPass.php

```diff
@@ -33,6 +36,14 @@ final class AttributeExtensionPass implements CompilerPassInterface
 
     public static function autoconfigureFromAttribute(ChildDefinition $definition, AsTwigFilter|AsTwigFunction|AsTwigTest $attribute, \ReflectionMethod $reflector): void
     {
+        $class = $reflector->getDeclaringClass();
+        if ($class->implementsInterface(ExtensionInterface::class)) {
+            if ($class->isSubclassOf(AbstractExtension::class)) {
+                throw new LogicException(\sprintf('The class "%s" cannot both extend "%s" and use the "#[%s]" attribute on method "%s()", choose one or the other.', $class->name, AbstractExtension::class, $attribute::class, $reflector->name));
+            }
+            throw new LogicException(\sprintf('The class "%s" cannot both implement "%s" and use the "#[%s]" attribute on method "%s()", choose one or the other.', $class->name, ExtensionInterface::class, $attribute::class, $reflector->name));
```

Comment:
```suggestion
            throw new LogicException(\sprintf('The class "%s" cannot implement "%s" and use the "#[%s]" attribute on method "%s()", choose one or the other.', $class->name, ExtensionInterface::class, $attribute::class, $reflector->name));
```

Reply:
suggested it in https://github.com/symfony/symfony/pull/60415?notification_referrer_id=NT_kwDOAHJ477MxNjM4ODUzOTc3MDo3NTAyMDYz#discussion_r2087701877 :) what's wrong with it?